### PR TITLE
Add tests for the operating_system field in system profile

### PR DESCRIPTION
**Pull request by ryandillinfelton**

### CODE REVIEW

**Review by kruai — APPROVED** [review]
LGTM!

**Review by thearifismail — COMMENTED** [review]
I think it needs one more test with name longer than 4 chars

File: tests/test_mq_service.py [hunk 1]
```diff
@@ -423,6 +423,27 @@ def test_add_host_externalized_system_profile(mq_create_or_update_host):
             mq_create_or_update_host(host_to_create)
 
 
+def test_add_host_with_operating_system(event_datetime_mock, mq_create_or_update_host, db_get_host):
+    """
+    Tests that operating_system in the system profile is ingested properly
+    """
+    operating_system = {"major": 5, "minor": 1, "name": "RHEL"}
```
[the remark this answers is not in the document]
Comment (ryandillinfelton) [reply tests/test_mq_service.py:430]: There is a negative test for the major version. I will expand it to test each of the subfields

**Review by thearifismail — APPROVED** [review]
LGTM

**Review by tahmidefaz — APPROVED** [review]
LGTM! :+1:

**Pull request merged**